### ISSUE TITLE
Incorrect warning on multiple attributes in one RDN

### ISSUE THREAD

**Issue by YuryStrozhevsky** [awslabs/certlint#64]
In your code you have [this warning](https://github.com/awslabs/certlint/blob/master/lib/certlint/namelint.rb#L121) on having "multiple attributes in one RDN" in one certificate. In fact I do not understand the warning and why you made it.

As from initial type definition we have this:
```asn.1
RDNSequence ::= Sequence OF RelativeDistinguishedName
		
RelativeDistinguishedName ::=
SET SIZE (1..MAX) OF AttributeTypeAndValue
```
And nothing stops us from having multiple `AttributeTypeAndValue` - could be `(1..MAX)` values inside one `RelativeDistinguishedName`.

So, could you describe why you made the warning in `certlint`?
